### PR TITLE
[TEMP] don't publish katello, katello-debug and katello-selinux on el9

### PR DESCRIPTION
**Pull request by evgeni**
these packages have unresolved dependencies, and thus would prevent repoclosure from succeeding

### CODE REVIEW

File: comps/comps-katello-el9.xml [hunk 1]
```diff
@@ -12,12 +12,12 @@
...
       <packagereq type="default">foreman-proxy-content</packagereq>
-      <packagereq type="default">katello</packagereq>
+      <!-- temp removal due to unresolved dependencies <packagereq type="default">katello</packagereq> -->
       <packagereq type="default">katello-client-bootstrap</packagereq>
       <packagereq type="default">katello-common</packagereq>
-      <packagereq type="default">katello-debug</packagereq>
+      <!-- temp removal due to unresolved dependencies <packagereq type="default">katello-debug</packagereq> -->
       <packagereq type="default">katello-repos</packagereq>
-      <packagereq type="default">katello-selinux</packagereq>
+      <!-- temp removal due to unresolved dependencies <packagereq type="default">katello-selinux</packagereq> -->
```
Comment (ekohl) [inline comps/comps-katello-el9.xml:20]: Is this the `candlepin-selinux` dependency?

Comment (evgeni) [reply comps/comps-katello-el9.xml:20]: Yes

**Review by ekohl — COMMENTED** [review]
:+1: to the approach of disabling things that aren't ready.

File: comps/comps-katello-el9.xml [hunk 2]
```diff
@@ -12,12 +12,12 @@
     <packagelist>
       <!-- katello core packages -->
       <packagereq type="default">foreman-proxy-content</packagereq>
-      <packagereq type="default">katello</packagereq>
+      <!-- temp removal due to unresolved dependencies <packagereq type="default">katello</packagereq> -->
       <packagereq type="default">katello-client-bootstrap</packagereq>
       <packagereq type="default">katello-common</packagereq>
-      <packagereq type="default">katello-debug</packagereq>
+      <!-- temp removal due to unresolved dependencies <packagereq type="default">katello-debug</packagereq> -->
```
[the remark this answers is not in the document]
Comment (evgeni) [reply comps/comps-katello-el9.xml:18]: would also need https://github.com/theforeman/puppet-foreman_proxy_content/pull/468 but yeah can do

[the remark this answers is not in the document]
Comment (evgeni) [reply comps/comps-katello-el9.xml]: otherwise done

**Review by ekohl — APPROVED** [review]

File: comps/comps-katello-el9.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (ekohl) [reply comps/comps-katello-el9.xml]: That module doesn't state it's compatible with EL9 anyway, so it's not a problem ;)

[the remark this answers is not in the document]
Comment (evgeni) [reply comps/comps-katello-el9.xml]: _yet_ ;)

**Pull request merged**